### PR TITLE
Bump deps

### PR DESCRIPTION
**Pull request by lnicola**

### CODE REVIEW

File: crates/ra_hir_ty/src/traits/chalk.rs [hunk 1]
```diff
@@ -183,6 +183,8 @@ impl<'a> chalk_solve::RustIrDatabase<Interner> for ChalkContext<'a> {
                     .collect(),
                 1,
             ),
+            // FIXME: fill these in
```
Comment (lnicola) [inline crates/ra_hir_ty/src/traits/chalk.rs:186]: r? @flodiebold

Comment (flodiebold) [reply crates/ra_hir_ty/src/traits/chalk.rs:186]: Return-position impl Trait can't have where clauses, and we don't support other kinds of opaque types yet, so this should be fine.

**Comment by flodiebold** [comment]
bors d+ (maybe you want to change the FIXME comment)

**Comment by lnicola** [comment]
bors r=flodiebold

**Pull request merged**